docker/Dockerfile.ci_cpu

```diff
@@ -83,3 +83,7 @@ RUN bash /install/ubuntu_install_caffe.sh
 # Github Arm(R) Ethos(TM)-N NPU driver
 COPY install/ubuntu_install_ethosn_driver_stack.sh /install/ubuntu_install_ethosn_driver_stack.sh
 RUN bash /install/ubuntu_install_ethosn_driver_stack.sh
+
+# Vitis-AI PyXIR CI deps
+COPY install/ubuntu_install_vai_packages.sh /install/ubuntu_install_vai_packages.sh
+RUN bash /install/ubuntu_install_vai_packages.sh
```

Comment:
suggestion: maybe `ubuntu_install_vitis_ai.sh` would be more descriptive?

Reply:
@leandron Yes, we will make it more descriptive. However, in the main Vitis-AI integration PR we add another script so it wouldn't be too clear if we call this ```ubuntu_install_vitis_ai.sh```. Also, as this script is only used for CI I think ```ubuntu_install_vitis_ai_packages_ci.sh``` would be better. What do you think?

Reply:
Hi could you explain the purpose of docker related files/scripts in the main integration PR? Ideally we should separate all environment related changes to this PR so that we could see the whole picture.

Reply:
@comaniac  The purpose of docker related files/scripts in the main integration PR is to build the docker image for Vitis-AI codegen infrastructure using Dockerfile.ci_vai and launching the docker container with necessary drivers if required. Should we add those files in this PR itself?

Reply:
> Also, as this script is only used for CI I think `ubuntu_install_vitis_ai_packages_ci.sh` would be better. What do you think?
I had a look on your other PR now, and I understand what you said about the other script. I think it would be beneficial to have all the CI change in one patch, if possible.

Reply:
@leandron The ci_vai docker we added in the other PR is actually not meant to be used for CI at the moment. It's the main docker we use in our flow as you can see in our documentation: https://github.com/apache/incubator-tvm/blob/8903b1a3251370ee1013fc2f9f3ef6004fa0e4b2/docs/deploy/vitis_ai.rst and builds on top of the general Vitis-AI docker (https://github.com/Xilinx/Vitis-AI) containing the necessary Vitis-AI tools for quantization, compilation, etc. 

Two questions: 
- Should we rename the ci_vai to make it more clear that it's our main docker environment and it's not really for CI at the moment?
- Should we add it to this PR even though it's meant to be used for CI?

Reply:
> Should we rename the ci_vai to make it more clear that it's our main docker environment and it's not really for CI at the moment?

I think it would be good to name it in a way that is obvious to people on what that Dockerfile is intended for. Looking at existing Dockerfiles (https://github.com/apache/incubator-tvm/tree/master/docker) I see there are three prefixes: `ci_*`, `conda_*` and `demo_`. Maybe from the repository point of view it fits more for a `demo` than `ci`? I'm not very sure about what the right answer here, maybe @tqchen will know better.

> Should we add it to this PR even though it's meant to be used for CI?

Even not being strictly related to CI, as @comaniac pointed out, it is related to environment changes (for example the change/fix on the `ubuntu_install_python.sh` that might affect other Dockerfiles). So I think it would be good to have them all in one PR.

Reply:
@leandron I am not sure about the clarity of the demo prefix either. Maybe we can just add it as Dockerfile.vitisai? @tqchen ?

And ok, we will move all environment changes to this PR.

Reply:
I agree with Leandro basically. In TVM we have two types of docker files: CI and Demo. Demo is used to provide an environment for users to quickly try out basic TVM functions. However, TVM does not provide the docker file for developments other than CI environment.

If you would like to provide an environment for TVM users that want to use Vitis-AI, the currently solution is either providing installation instructions in the tutorial or document like ACL does to help users set up the environment, or make it as a Demo image and point users to there.

Of course, ideally it would be better for TVM to have a set of docker files for developments, but it is hard to achieve for some backends due to the license issue.

cc @tqchen for more comments.

Reply:
@comaniac 
I have added Dockerfile.demo_vitis_ai and corresponding environment related scripts. Could you please review those?

Reply:
LGTM @leandron lemme know if you're happy with the outcome of the file renaming.

Reply:
Thanks @tmoreau89 - yes, it looks more integrated with existing Dockerfiles now. LGTM